code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -114,6 +114,8 @@ var/global/list/datum/stack_recipe/wood_recipes = list ( \
 	new/datum/stack_recipe("dog bed", /obj/structure/bed/dogbed, 10, time = 10, one_per_turf = 1, on_floor = 1), \
 	new/datum/stack_recipe("display case chassis", /obj/structure/displaycase_chassis, 5, one_per_turf = 1, on_floor = 1), \
 	new/datum/stack_recipe("wooden buckler", /obj/item/weapon/shield/riot/buckler, 20, time = 40), \
+	new/datum/stack_recipe("apiary", /obj/structure/beebox, 40, time = 50),\
```

Comment:
I have no idea how much wood the average towercap yields, but this seems very high vs the other options